diesel_tests/tests/expressions/date_and_time.rs

```diff
@@ -333,7 +428,7 @@ fn adding_interval_to_nullable_things() {
     assert_eq!(expected_data, actual_data);
 }
 
-#[cfg(not(feature = "mysql"))] // FIXME: Figure out how to handle tests that modify schema
+#[cfg(any(feature = "postgres", feature = "sqlite", feature = "mysql"))]
```

Comment:
Can you explain how you addressed the comment about tests modifying the schema on mysql? In detail: How does this prevent failing tests for cases where tables already exist?

Reply:
I'm sorry. I removed the comment by mistake.

Reply:
While that adds back the comment it also allows this function to run on mysql, therefore it does not really address my remark above.

Reply:
I've added this comment again!

Reply:
Again: It's not about the comment, it's about the behavior. If you try to run the tests twice against a mysql database they will fail because tables already exists.

Reply:
Ok, I misunderstood.